.php_cs

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
This file is taken from AdminUI: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/.php_cs